RLBotPack/Noob_Black/src/bot.cfg

```diff
@@ -0,0 +1,33 @@
+[Locations]
+# Path to loadout config. Can use relative path from here.
+looks_config = ./appearance.cfg
+
+# Path to python file. Can use relative path from here.
+python_file = ./bot.py
+
+# Name of the bot in-game
+name = Noob Black
+
+# The maximum number of ticks per second that your bot wishes to receive.
+maximum_tick_rate_preference = 120
+
+[Details]
+# These values are optional but useful metadata for helper programs
+# Name of the bot's creator/developer
+developer = Scumclass
+
+# Short description of the bot
+description = Noob Bot V3, made with code from the black market.
+As always, mainly designed for 3s.
+
+# Fun fact about the bot
+fun_fact = The bot has a Roblox background.
+
+# Tags
+tags = goalie, teamplay, rumble, hoops

```

Comment:
The goalie tag is reserved for bots that ONLY play goalie. The rumble tag is for bots that use the rumble items.